src/Resources/config/services.yml

```diff
@@ -1,7 +1,7 @@
 services:
     partnermarketing_file_system.factory:
-        class: "Partnermarketing\FileSystemBundle\Factory\FileSystemFactory"
+        class: "Partnermarketing\\FileSystemBundle\\Factory\\FileSystemFactory"
         arguments:
-            - %partnermarketing_file_system.default_file_system%
-            - %partnermarketing_file_system.config%
-            - %partnermarketing_file_system.tmp_dir%
+            - "%partnermarketing_file_system.default_file_system%"
```

Comment:
Unquoted scalar values starting with '%' for parameters was deprecated in 3.1, to be removed in 4.0. You are correct that they're not strictly required right now, but whilst I'm here, might as well clean up deprecation warnings.